examples/website/3d-heatmap/app.js

```diff
@@ -164,19 +147,22 @@ class App extends Component {
     return (
       <DeckGL
         layers={layers}
-        views={new MapView({id: 'map'})}
+        initialViewState={INITIAL_VIEW_STATE}
         viewState={viewState}
         onViewStateChange={onViewStateChange}
         controller={MapController}
       >
-        <StaticMap
-          viewId="map"
-          {...viewState}
-          reuseMaps
-          mapStyle={mapStyle}
-          preventStyleDiffing={true}
-          mapboxApiAccessToken={mapboxApiAccessToken}
-        />
+        {({viewState: mapViewState, width, height}) => (
```

Comment:
I know these are just nits but:

* The `viewState` renaming to get around lint is really annoying, detracts from the example. Sigh...
* Also the need to pass around `width` and `height` adds back clutter we have fought hard to remove (and makes it painfully obvious that react-map-gl doesn't support auto-resize).